megamek/src/megamek/client/ui/swing/minimap/Minimap.java

```diff
@@ -536,6 +536,9 @@ private void drawMap(boolean forceDraw) {
             for (int j = 0; j < board.getWidth(); j++) {
                 for (int k = 0; k < board.getHeight(); k++) {
                     Hex h = board.getHex(j, k);
+                    if (h == null) {
+                        continue;
+                    }
```

Comment:
Missing nullpointer check. It should never be null, but had it come back null to me a couple of times...